doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+# Multi Orchestrator Setup
+
+This document describes the various ways an orchestrator could run multiple nodes and separate concerns. 
+
+## Unlocking Account and Connecting to Ethereum
+
+Unlocking your Ethereum account and connecting to a JSON-RPC provider are necessary steps to run the Livepeer node unless it's in standalone transcoder mode. 
+
+For brevity we'll exclude these flags in the examples by `<...ETH SETUP...>` but these should always be required. If no Ethereum account is available the Livepeer node will create one for you.
+
+```
+livepeer
+    -network <mainnet|rinkeby> \
+    -ethUrl <JSON_RPC_PROVIDER> \
+    -ethKeystorePath <PATH_TO_KEY> \
+    -ethAcctAddr <ETHEREUM_ADDRESS> \
+    -ethPassword <ETHEREUM_ACCOUNT_PASSWORD (string|path)> \
+    -ethController <CONTROLLER_CONTRACT_ADDRESS (required if '-network' not provided)>
+```
+
+## Single Orchestrator with Redeemer
+
+This allows an orchestrator running a single on-chain node to use a seperate Ethereum account to redeem winning tickets on-chain and pay for that transaction while the recipient is still orchestrator node's Ethereum address.
+
+The orchestrator node will still be responsible for initialising rounds and calling `reward`. 
+
+1. Start the Redeemer 
+
+```shell
+livepeer
+    <...ETH SETUP...\
+    -redeemer=true \ 
+    -httpAddr <REDEEMER_HTTP_ADDR (host):port \ 
+    -ethOrchAddr <ORCHESTRATOR_ON_CHAIN_ETH_ADDR (also the recipient address)>
+```
+
+2. Start the Orchestrator
+
+
+```shell
+livepeer
+    <...ETH SETUP...> \
+    -orchestrator=true -transcoder=true \
+    -initializeRound=true \
```

Comment:
Orchestrators aren't required to initialize rounds right now which is why the round initializer service is off by default and we don't communicate that orchestrators should be running with this flag right now. What do you think about excluding the round initialization bit from these docs right now and only including it when we're communicating that the flag should be used by orchestrators regardless of what kind of setup they are using?

Reply:
I think we should avoid a scenario whereby people feel comfortable omitting this flag, so how about the reverse ? 

Keep it in but mention that livepeer currently 'sponsors' round initialisation but that ideally node operators shouldn't rely on single points of failures like that and make sure they can do everything within their means to run robust infra.

Reply:
I hear you on wanting orchestrator operators to expect to initialize rounds. But, it is worth noting that we currently do not indicate anywhere in our docs that the `-initializeRound` flag should be used. So, it feels a bit weird to drop in this flag in this specific doc which is focused on multi-O setups when the flag is not mentioned anywhere else. I'd be more inclined to mention the `-initializeRound` flag in a more general purpose doc first along with a description of what it does before including it in this doc.